specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2801,7 +2801,12 @@
             "UrlFileName",
             "HttpVersion",
             "Cookies",
-            "IsDevice"
+            "IsDevice",
+            "SocketAddress",
+            "ClientPort",
+            "ServerPort",
+            "Hostname",
```

Comment:
Should we use HostName to be consistent?

Reply:
updated casing